main.js

```diff
@@ -188,6 +188,19 @@ module.exports = function(options) {
 
 	let flagsPromise = Promise.resolve();
 
+	app.use((req, res) => {
+		setTimeout(() => {
+			nLogger.error('RESPONSE_TIMED_OUT', {
+				url: req.url
+			});
+			try {
+				res.sendStatus(504)
+			} catch (e) {
+				// avoids sending a 'headers already sent' error to sentry
```

Comment:
also can this be an optional feature for now while we test it in a non-platinum app?